index.js

```diff
@@ -147,22 +155,42 @@ const decoder = {
     const tf = buf.slice(0, 2)
     const d = buf.slice(2)
 
-    let msgextension = ''
+    let msgextension
     if (tf.equals(CLASSIC_MSG_TF)) msgextension = '.sha256'
-    else if (tf.equals(BB_MSG_TF)) msgextension = '.bbmsg-v1'
-    else if (tf.equals(GG_MSG_TF)) msgextension = '.ggmsg-v1'
+    else if (tf.equals(GABBYGR_MSG_TF)) msgextension = '.ggmsg-v1'
+    else if (tf.equals(BENDYBT_MSG_TF)) msgextension = '.bbmsg-v1'
     else throw new Error('Unknown msg: ' + buf)
 
     const b64part = d.toString('base64')
     return '%' + b64part + msgextension
   },
 
+  blob(buf) {
+    const tf = buf.slice(0, 2)
+    const d = buf.slice(2)
+
+    let blobext
```

Comment:
Another minor style comment for consistency and readability:

maybe `blobExtension` instead of `blobext`? And then `feedextension` and `msgextension` also as camel case: `feedExtension` and `msgExtension`?

Or at least `blobext` -> `blobextension` for consistency across the three.